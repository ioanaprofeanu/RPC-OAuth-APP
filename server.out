DENY (READ,UserData,,0)
BEGIN X5B0TWjmeNtU3vd AUTHZ
  RequestToken = v53teNBmWTXjUd0
  AccessToken = mNBIv3N_5DSeGEt
PERMIT (READ,UserData,mNBIv3N_5DSeGEt,1)
DENY (DELETE,Preferences,mNBIv3N_5DSeGEt,0)
DENY (EXECUTE,UserData,mNBIv3N_5DSeGEt,-1)
BEGIN X5B0TWjmeNtU3vd AUTHZ
  RequestToken = dvBj3mU5WXTtN0e
  AccessToken = 5U_GNSDjBv3IEdm
DENY (MODIFY,SystemData,5U_GNSDjBv3IEdm,1)
PERMIT (EXECUTE,Applications,5U_GNSDjBv3IEdm,0)
BEGIN QHmJh2xwErPDcmv AUTHZ
  RequestToken = mxvJQE2whHrcDPm
BEGIN PYM3XptJ6asHaRp AUTHZ
  RequestToken = pPHXta6YpMas3RJ
  AccessToken = S_ENG6pHtDaIXYP
DENY (EXECUTE,Random,S_ENG6pHtDaIXYP,1)
