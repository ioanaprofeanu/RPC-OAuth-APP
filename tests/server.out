BEGIN 74xaLqdTDdHZ9ey AUTHZ
  RequestToken = e49HDdxTqL7dZya
  AccessToken = aZy4TLHxqded9D7
BEGIN f80N46i82fdx61d AUTHZ
  RequestToken = d10i68x86f4dfN2
  AccessToken = 2Nx0id84f6d168f
  RefreshToken = x8f02fd4iN661d8
PERMIT (READ,Applications,2Nx0id84f6d168f,1)
PERMIT (EXECUTE,Applications,2Nx0id84f6d168f,0)
DENY (DELETE,Applications,,0)
PERMIT (READ,Preferences,aZy4TLHxqded9D7,1)
DENY (MODIFY,Preferences,aZy4TLHxqded9D7,0)
DENY (MODIFY,UserData,,0)
BEGIN 74xaLqdTDdHZ9ey AUTHZ
  RequestToken = q7ZLd9D4yxdHaeT
DENY (INSERT,Files,,0)
DENY (INSERT,Applications,,0)
BEGIN f80N46i82fdx61d AUTHZ
  RequestToken = fi4N018fx668dd2
  AccessToken = x1628id60N8ffd4
  RefreshToken = 48d2606dx1fNi8f
PERMIT (INSERT,Applications,x1628id60N8ffd4,1)
DENY (DELETE,Files,x1628id60N8ffd4,0)
DENY (EXECUTE,Applications,,0)
DENY (DELETE,Applications,,0)
